perf-tool/src/monitor.cpp

```diff
@@ -54,6 +55,18 @@ JNIEXPORT void JNICALL MonitorContendedEntered(jvmtiEnv *jvmtiEnv, JNIEnv *env,
 	    j["monitorHash"] = str;
     }
 
+    std::string h = std::to_string(hash);
```

Comment:
Why convert the hash to string? Isn't easier to use it directly?

Reply:
type mismatch will occur if we use it directly so I converted into `string`